src/components/career-form/career-form.tsx

```diff
@@ -60,14 +82,20 @@ export const CareerForm: React.FC<CareerFormProps> = (props) => {
 
     for (const [key, value] of Object.entries(payload)) {
       if (key === 'documents') {
-        formData.append(
-          'documents',
-          formValues.documents[0],
-          formValues.documents[0].name,
-        );
+        for (let i = 0; i < formValues.documents.length; i++) {
+          const keyName = `categorised_documents[${i}][file]`;
+          const fileName = formValues.documents[i].name.split('.')[0];
+          const category = formValues.category_select[fileName];
+          formData.append(keyName, formValues.documents[i]);
+          const nameCategory = `categorised_documents[${i}][category]`;
```

Comment:
nice that you figured out the personio api format. I don't like their API description and it's difficult to find the correct combination but you succeeded here👍